nngt/plot/plt_properties.py

```diff
@@ -72,9 +71,24 @@ def degree_distribution(network, deg_type="total", nodes=None,
         created.
     show : bool, optional (default: True)
         Show the Figure right away if True, else keep it warm for later use.
+    title : string, optional title
     **kwargs : keyword arguments for :func:`matplotlib.axes.Axes.bar`.
     '''
     import matplotlib.pyplot as plt
+
+    #More modern look
+    plt.rcParams['font.family'] = 'serif'
+    plt.rcParams['font.serif'] = 'Ubuntu'
```

Comment:
Do not use non-default fonts, this font is not available on all platforms (it's even not available on all Linux platforms)